test/run_pass/issue/issue-133.js

```diff
@@ -13,6 +13,14 @@
  * limitations under the License.
  */
 
+if (this.driver) {
+  var originalConsole = console;
+  console = require('tools/commonJS/modules/console_wrapper');
```

Comment:
Absolutely not, I just meant this is common modules for javascript. just like common direcotry for python. How about renaming 'common_js' and 'common_py' then?